griptape/loaders/base_file_loader.py

```diff
@@ -27,3 +27,9 @@ def fetch(self, source: str | PathLike) -> bytes:
             return data.encode(self.encoding)
         else:
             return data
+
+    def save(self, destination: str | PathLike, artifact: A) -> None:
+        """Saves the Artifact to a destination."""
+        data = artifact.value if isinstance(artifact.value, bytes) else artifact.value.encode(self.encoding)
+
+        self.file_manager_driver.save_file(str(destination), data)
```

Comment:
Surprised `save_file` doesn't take `PathLike` too.

Reply:
Probably should be. Would prefer to keep that change out out of this PR though.